test/integration/query.spec.ts

```diff
@@ -30,8 +30,7 @@ describe("ResourceLink Trimming of leading and trailing slashes", function() {
 
     await container.items.create(doc);
     const query = "SELECT * from " + containerId;
-    const queryOptions = { partitionKey: "pk" };
-    const queryIterator = container.items.query(query, queryOptions);
+    const queryIterator = container.items.query(query);
```

Comment:
You might already have this but can you assert that unsupported queries are not allowed:

like GROUP BY

Reply:
I added a test for GROUP BY erroring in the aggregates file